test/unit/test_cublas_tile/test_trmm.h

```diff
@@ -0,0 +1,87 @@
+//
```

Comment:
No need to duplicate this test. See the other tests for device generic implementations.